frontend/src/cljs/imas_seamap/map/events.cljs

```diff
@@ -159,32 +150,54 @@
 (defn toggle-ignore-click [db _]
   (update-in db [:map :controls :ignore-click] not))
 
-(defn got-feature-info [db [_ request-id priority point info-format response]]
+(defn display-feature-info [db point]
+  (let [responses      (get-in db [:feature-query :responses])
+        response-infos (map
+                        (fn [response]
+                          (if response
+                            (let [{:keys [response info-format]} response]
+                              (case info-format
+                                "text/html" (feature-info-html response)
+                                "application/json" (feature-info-json response)
+                                (feature-info-none)))
+                            {:status :feature-info/empty}))
+                        responses)
```

Comment:
So, a few things here:

- My eyes jumped to this block of code (and response-info below); it's quite blocky and not super readable.  Any time you have an anonymous function in a map or (particularly) reduce, if it's more than a couple of lines it attracts attention I find.
- https://clojuredocs.org/clojure.core/if-let :)
- In general I think I'd write functions to at least name what you're doing (https://clojuredocs.org/clojure.core/letfn might be useful too)